src/renderer/features/governance/components/ReferendumDetails/ProposalDescription.tsx

```diff
@@ -15,6 +16,8 @@ type Props = {
 
 export const ProposalDescription = ({ chainId, referendum }: Props) => {
   const isDescriptionLoading = useUnit(detailsAggregate.$isDescriptionLoading);
+  const addressPrefix = useUnit(networkModel.$chains)[chainId]?.addressPrefix;
```

Comment:
In governance we have `networkSelectorModel.$governanceChain`, can you use it instead?

Reply:
absolutely, thanks.